docs/testing.md

```diff
@@ -0,0 +1,68 @@
+# Testing
+
+We want to maintain a project with a high coverage (aiming for 100%). Our main [coverage criterion](https://en.wikipedia.org/wiki/Code_coverage#Coverage_criteria) is **branch coverage**, that is making sure all possible code branches are covered by test. [Coveralls](https://coveralls.io/github/mozilla/addons-frontend) helps us monitor the status of our test suite coverage and it is a good idea to double-check its output when you are opening a Pull Request. You can also [generate a coverage report locally](https://github.com/mozilla/addons-frontend/#code-coverage).
+
+## General guidelines
+
+- Imports should be alphabetized, even in test file.
+- Comments should be full sentences to ease readability.
+- There are a lot of helpers in the `tests/unit/helpers` and `tests/unit/amo/helpers` modules.
+- Use the action creators instead of hard coding props. This applies to both UI components and reducers/sagas.
```

Comment:
"Use action creators in `amo/actions/` and `core/actions/` instead of hard-coding `dispatch()` arguments or state data for tests. This applies to both UI components and sagas."

(We don't dispatch from reducers so I don't think it matters there, unless you meant for tests... in which case I suppose you can leave it in.)

Reply:
I actually had to use an action creator to init a `state`.